### ISSUE TITLE
Zoom option for region maps

### ISSUE THREAD

**Issue by ignacio-mb** [metabase/metabase#33928]
**Is your feature request related to a problem? Please describe.**
Zoom In/out options for Region maps

Region map
<img width="983" alt="Screenshot 2023-09-18 at 4 49 06 PM" src="https://github.com/metabase/metabase/assets/132273646/fa2f77ba-c426-4d61-8033-2731500a6cc9">

Pin map
<img width="779" alt="Screenshot 2023-09-18 at 4 49 52 PM" src="https://github.com/metabase/metabase/assets/132273646/a563a564-d13b-42ed-9064-f8b578effb36">

Grid map
<img width="785" alt="Screenshot 2023-09-18 at 4 51 45 PM" src="https://github.com/metabase/metabase/assets/132273646/0c5e30cd-479a-4183-acc6-36fa3ed11b1d">

**Describe the solution you'd like**
Same as Grid and Pin maps offer 

**Describe alternatives you've considered**
Using grid map but not the same

**How important is this feature to you?**
Requested by a customer. Useful for use cases where the regions are in different parts of the world

**Additional context**
Is there any specific reason this map doesn't have the zoom feature?

**Comment by mazameli** [comment]
Like, literally just zoom in on the region map? Is the user/customer dealing with really small regions in a custom geojson map file or something?

**Comment by ignacio-mb** [comment]
Exactly, a zoom mechanism. Small countries like Monaco or Luxembourg can't be reached from the "World" view (which can be a separate Feature request, as they currently aren't part of the World map list of countries)

For reference of the request, internal ticket # 20817

**Comment by talenodigital** [comment]
We are using the Region Map for Norway as well. Is there a Feature Request to have Zoom In / Out on Region Map?

**Comment by stormmaster42** [comment]
Another example for this here: we use a region map of  ZIP codes in Germany.
Would be nice to be able to zoom in on the relevant areas.
![Image](https://github.com/user-attachments/assets/87271f79-9e04-43da-9135-56f534c14084)

**Comment by DenisBalan** [comment]
Also if you rewrite the zoom/map module
Please add ability to zoom in/out by mouse scroll, not buttons only, applicable on all map types.

**Comment by CharlesRngrd** [comment]
Also facing the same missing zoom.
Here is my map. As you can see, there are many areas in Paris but there are quite unvisible :

<img width="1440" height="855" alt="Image" src="https://github.com/user-attachments/assets/72ad9a66-2052-4162-8e01-5430f0af5d56" />

Could you plan the zoom feature please ?

PS: This popular feature request could be a good alternative https://github.com/metabase/metabase/issues/4724
